tools/go.mod

```
@@ -1,6 +1,6 @@
 module github.com/stackrox/acs-fleet-manager/tools
 
-go 1.23.0
+go 1.24
```

It is preferable to use stricter semantics, introduced in Go 1.21.
```suggestion
go 1.24.0
```